### PR TITLE
sys-app/ignition: add initial ebuild and service

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: sys-apps/ignition/files/ignition@.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+After=initrd-fs.target
+ConditionPathExists=/etc/initrd-release
+ConditionPathExists=!/sysroot/etc/machine-id
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+Environment=OEM_NAME=pxe
+EnvironmentFile=-/usr/share/oem/environment
+# TODO(crawford): /sysroot should be a variable
```
Comment (marineam) [inline sys-apps/ignition/files/ignition@.service]: eh, /sysroot is hard coded into systemd ;-)

File: sys-apps/ignition/files/ignition@.service [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ExecStart=/bin/ignition --root=/sysroot --oem=${OEM_NAME} --stage=%i
```
Comment (marineam) [inline sys-apps/ignition/files/ignition@.service]: Dracut doesn't put it in /usr/bin like it is on the filesystem?

Comment (crawford) [inline sys-apps/ignition/files/ignition@.service]: Fixed :)

**Comment by marineam** [comment]
Drop the pointless TODO, /sysroot is always the same in the initrd. Otherwise LGTM

[LGTM](http://lgtm.in/i/KFdvznpiq)

**Pull request merged**